samples/README.md

```diff
@@ -0,0 +1,83 @@
+2. *Include Metadata* as specified in [`datapackage.json`](#data-package).
+3. *Include a README.md* in the base folder of your example with an overview so that it can be included in the documentation.
+
+## Data Package
+
+TIDES data packages must include a [`datapackage.json`](https://specs.frictionlessdata.io/data-package/).  Key information to include in [`datapackage.json`](https://specs.frictionlessdata.io/data-package/) includes:
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `title` | A human-readable title. | Required |
+| `name` | Identifier string as a URL-friendly slug. | Required |
+| `description` | Short description of data package. | Recommended |
+| `agency` | Transit agency name. | Recommended |
+| `ntd_id` | ID for the National Transit Database. | Recommended |
+| `profile` | Should be `tabular-data-package` | Required |
+| `licenses` | Should be `[{"name": "Apache-2.0"}]` to be consistent with this repository | Required |
+| `contributors` | Array of data contributors `[{"title": "My Name", "github": "my_handle", "email": "me@myself.com"}]` | Recommended |
+| `maintainers` |  Array of data maintainers `[{"title": "My Name", "github": "my_handle", "email": "me@myself.com"}]` | Recommended |
+| `resources` | Array of data files included in your package, formated as a [`tabular-data-resource`](#data-resource)| Required |
+
+### Template
+
+A `datapackage.json` template is available at [`/data/template/TIDES/datapackage.json`](https://raw.githubusercontent.com/TIDES-transit/TIDES/main/data/template/TIDES/datapackage.json).
+
+Once `datapackage.json` is created for your data, you can easily conduct [data validation](#data-validation) using a variet of tools.
+
+### Data Resource
+
+Key fields for each [`tabular-data-resource`](https://specs.frictionlessdata.io/tabular-data-resource/) are as follows:
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `name` | Short sluggable name used to refer to data in this file. | Required |
+| `path` | Path of the data resource file relative to the `datapackage.json` | Required |
+| `schema` | Data schema to use to valdiate the data resource to | Required |
+| `sources` | Array of data sources formatted as a [`source`](#data-source) | Recommended |
+
+### Data Source
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `title` | Description of the data source. | Required |
+| `component` | What technology component was used to generate this data (directly or indirectly)? Examples include `AVL`, `APC`, `AFC`, etc.  | Recommended |
+| `product` | What product was used to generate this data (directly or indirectly)? | Recommended |
+| `vendor` | What company makes this product? | Recommended |
```

Comment:
The main value I see in putting it under each resource is to explicitly say where each piece of the "TIDES pie" is coming from so that agencies that are trying to mimic another agency's a data flow diagram can do so.  It's not necessarily implicit which resource comes from which product.  For example, there are *many* agencies that receive APC data from their "APC product" that is then manipulated by another product (including internal scripts) and then spat out to other systems.  Exposing this seems like it would add a great deal of value.  That said, there is no reason why you *can't* just put all the sources up top....especially if we aren't articulating a profile which would require it and are merely suggesting.

Reply:
Sure. It could be nested, big-picture at the top-level, details for each resource, both optional.